TypeCobol.Test/Utils/Comparator.cs

```diff
@@ -466,7 +472,63 @@ internal void Compare(IEnumerable<Program> programs, IEnumerable<TypeCobol.Compi
             ParserUtils.CheckWithResultReader(paths.SamplePath, result, expected, expectedResultPath);
         }
     }
+ 
+    internal class SqlComparator : FilesComparator
+    {
+        /// <summary>
+        /// Selects the SQL code elements
+        /// </summary>
+        private class ASTVisitor : AbstractAstVisitor 
+        {
+            private readonly StringWriter _writer;
+            public ASTVisitor(StringBuilder builder)
+            {
+                _writer = new StringWriter(builder);
+            }
+
+            public override bool Visit(SelectStatement selectStatement)
+            {
+                _writer.WriteLine("line " + selectStatement.Line + ": SelectStatement");
+                selectStatement.FullSelect.Dump(_writer ,1);
+                return true;
+            }
+
+            public override bool Visit(CommitStatement commitStatement)
+            {
+                _writer.WriteLine("line" + commitStatement.Line + ":CommitStatement");
```

Comment:
Add a space after 'line' and after ':' to match the syntax used for SelectStatement